app/hooks/useSettingsConfig.ts

```diff
@@ -183,6 +193,15 @@ const useSettingsConfig = () => {
         group: t("Integrations"),
         icon: ZapierIcon,
       },
```

Comment:
I'd rather not have Zapier as a special case here, it can be on the index page with the others